extensions/amp-access/0.1/amp-access-vendor.js

```diff
@@ -42,8 +42,8 @@ export class AccessVendorAdapter {
     this.vendorName_ = user().assert(configJson['vendor'],
         '"vendor" name must be specified');
 
-    /** @const @private {JsonObject} */
-    this.vendorConfig_ = configJson[this.vendorName_];
+    /** @const @private {!JsonObject} */
+    this.vendorConfig_ = configJson[this.vendorName_] || {};
```

Comment:
I spent a good amount of time w/ this code recently and also just did a second audit....I think the only user of vendor before us required a config so this was never null in practice. This issue actually arose because although the definition of `getConfig()` is to return `!JsonConfig` the type checking did not seem to be applied here through the `@override` of that method here. i.e. it should never have been possible to write this code in the first place. That said, totally happy for dima to 2x check.

Reply:
@dvoytenko could you take a quick look?